pysmt/smtlib/interpolator.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of pySMT.
+#
+#   Copyright 2014 Andrea Micheli and Marco Gario
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import pysmt.smtlib.commands as smtcmd
+from pysmt.exceptions import SolverReturnedUnknownResultError, UnknownSolverAnswerError
+from pysmt.smtlib.script import SmtLibCommand
+from pysmt.smtlib.solver import SmtLibSolver
+from pysmt.solvers.interpolation import Interpolator
+
+
+class SmtLibInterpolator(Interpolator):
+
+    def __init__(self, args, environment, logic, LOGICS=None, **options):
+        Interpolator.__init__(self)
+        options["produce_interpolants"] = True
+        self._solver = SmtLibSolver(args, environment, logic, LOGICS, **options)
+
+    def binary_interpolant(self, a, b):
```

Comment:
This is a new interface. Could you add a comment on both `binary_interpolant` and `sequence_interpolant` on how to use them?